platforms/defaults_windows_test.go

```diff
@@ -204,9 +206,11 @@ func TestMatchComparerMatch_LCOW(t *testing.T) {
 }
 
 func TestMatchComparerLess(t *testing.T) {
+	ubr := getHostWindowsUpdateBuildRevision()
 	m := windowsmatcher{
 		Platform:        DefaultSpec(),
 		osVersionPrefix: "10.0.17763",
+		osUBR:           ubr,
```

Comment:
I think it would be better to use a fixed `ubr` here, like we do for `osVersionPrefix`, to make it obvious that the `Less` implementation depends on the `osUBR` value, not the current host's UBR.